docs/pages/operator/init-manifests.mdx

```diff
@@ -17,4 +18,56 @@ init:
     apiVersion: v1
     kind: ConfigMap
     ...
-```
\ No newline at end of file
+```
+
+## Applying charts on vcluster initialization
+vcluster now supports applying helm charts while initializing a new vcluster. Currently 2 methods of applying charts are supported:
+1. Upstream Mode
+2. Bundle Mode
+
+#### Upstream Mode
+This is the most straight forward approach of applying a helm chart existing in and public/private upstream chart repository. the following examples demonstrate the usage:
```

Comment:
`the following examples demonstrate the usage:`
Please capitalize the first word here.